content/participate/report-issue.adoc

```diff
@@ -0,0 +1,215 @@
+haven't already. Then, just select *Create Issue* on
+https://issues.jenkins-ci.org/secure/Dashboard.jspa[the JIRA home page].
+
+* For *Project* select:
+** _Jenkins_ for general issues with Jenkins,
+** _Security Issues_ if you want to report a security issue privately,
+or
+** _Infrastructure_ if you're reporting a bug with the Jenkins web site,
+wiki, or any other services run by the Jenkins project.
+* Enter a short but meaningful description of your problem as *Summary*.
+* For *Priority*, see
+https://issues.jenkins-ci.org/secure/ShowConstantsHelp.jspa?decorator=popup#PriorityLevels[here]
+for descriptions of the levels. *This is not how urgently* *_you_* *want
+it fixed*. Try to be realistic: Most bugs should be Minor or Major
+priority. If you found a workaround, reduce the priority, even if it's a
+severe problem otherwise.
+* As *Component*, select _core_ if it's a problem with Jenkins itself
+and not any of the plugins. Otherwise, select the name of the plugin;
+all of them should have a related component. You can also select
+multiple components e.g. if the problem is in the interaction of the
+components, or when you're not sure.
+* Leave the *Assignee* field to automatic.
+
+[[Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription]]
+=== What information to provide for Environment and Description
+
+[.aui-icon .aui-icon-small .aui-iconfont-warning .confluence-information-macro-icon]#
+#
+
+Much of the advice on
+http://www.catb.org/esr/faqs/smart-questions.html[How To Ask Questions
+The Smart Way] also helps you write great issue reports.
+
+*Environment* is mostly relevant for bug reports: What is the
+configuration experiencing the issue? Depending on the type of problem,
+the following are often relevant:
+
+* *Operating System* (including version, and whether it's 32/64 bit) of
+all systems involved (your client's, the Jenkins server's, all slave
+nodes'),
+* All relevant *JRE/JDK vendors and versions* (e.g. Oracle JRE, OpenJDK,
+...) and the parameters set.
+* *Jenkins and plugin versions*,
+* Whether you're *running Jenkins directly or in a container* like
+Tomcat (which one, in which version?)
+* Whether Jenkins is accessed through a *reverse proxy* (if so, how is
+it configured?)
+* *How you installed Jenkins* (Windows installer, deb/rpm, ...), and
+*how you're launching any involved slave nodes* (via SSH, web browser,
+command line, ...)
+* *Your web browser* (+ version). Make sure the issue occurs on
+https://wiki.jenkins.io/display/JENKINS/Browser+Compatibility+Matrix[supported
+web browsers].
+
+If this is too much information for the Environment field, just put some
+of it into the description.
+
+Every Jenkins installation has the `+/systemInfo+` URL with information
+about Jenkins. If you're not sure what information to provide, copying
+everything on that page is a good place to start. Another option would
+be to install the
+https://wiki.jenkins.io/display/JENKINS/Support+Core+Plugin[Support Core
```

Comment:
```suggestion
link:https://plugins.jenkins.io/support-core/[Support Core
```

Reply:
replaced wiki page with link to plugins site